packages/core/src/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from './CommandManager.js'
+export * from './commands/index.js'
```

Comment:
May be worth namespacing this since it will be a number of commands that people might think they can use without the editor instance:

```suggestion
export * as commands from './commands/index.js'
```

Also reduces the long list of exports you'd see on the package